### ISSUE TITLE
How do I use the additional arguments?

### ISSUE THREAD

**Issue by brunocastello** [utmapp/UTM#75]
I tried to use them to change the hw for sound and vga, but it didn’t worked. I tried to use them like it’s done with qemu on terminal.

I have UTM on my iPad Pro 1st gen 12.5, 64GB. I am trying with a Windows 2000 VM, I managed to install the OS and get to the desktop, networking kinda works, but there are a few devices with an yellow exclamation mark on my device manager, no sound and vga is limited to 16 colors / 800x600. Also, I have a smart keyboard cover, but the app does not recognize my keyboard input (only the virtual one on screen). I have a PCI device, unknown device, and VGA device without proper drivers.

Meanwhile, my dad is trying the same on his iPad (same as mine) but with Ubuntu 16.04 and he failed to install. I was lucky to go even further with Win 2000.

I have to ask if someone could point me to the right direction so I can make a better use of my favorite OS? I have the same OS on my Mac with vmware fusion, working 100%. It’d be great to have it on my iPad - I need to sell my 2012 MacBook and save money for a newer one.

**Comment by osy86** [comment]
There is an option in the configs to add additional args. But if you want specific options to be in the GUI config, please suggest them.

**Comment by brunocastello** [comment]
> There is an option in the configs to add additional args. But if you want specific options to be in the GUI config, please suggest them.

Thanks, but this exactly what I am talking about. That option. I had added the args I wanted to use, but it didn’t change anything. Hence why I came to ask about how to use the args in that option. For example with qemu I’d have used “-vga cirrus -soundhw sb16” without the double quotes, ofc.

As for the specific options, the Display and Sound menu tabs does not have an option to choose which hardware I want to use (for example Sound, choose SB16 or something else, like AC97).

Under System tab, the option to select a System is empty. I can make a screenshot if needed later. I’m on iPhone and the UTM is on my iPad Pro.
